dockerfile_functions.sh

```diff
@@ -604,6 +604,23 @@ print_cmd() {
 	fi
 }
 
+copy_scc_script() {
+    if [ "${vm}" == "openj9" ]; then
+        cat >> "$1" <<-EOI
+COPY generate_openj9_scc.sh /scripts/
+EOI
+    fi
+}
+
+run_scc_gen() {
+    if [ "${vm}" == "openj9" ]; then
+        cat >> "$1" <<-EOI
+        RUN /scripts/generate_openj9_scc.sh
+        ENV OPENJ9_JAVA_OPTIONS="-Xshareclasses:name=default_scc,cacheDir=/opt/java/.scc,bootClassesOnly,nonFatal"
```

Comment:
this ENV setting will be used by applications that are run with this container, right? If I understood that correctly, then I think we should take out `bootClassesOnly` and include `readyOnly` since not much else could be written into the default cache anyway (because of the way it's sized).